### PR TITLE
Rails 5 compatibility

### PR DESCRIPTION
**Pull request by bquorning**

### CODE REVIEW

**Comment by bquorning** [comment]
cc @morten @grosser

**Comment by bquorning** [comment]
@pschambacher

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
   - gemfiles/rails_4.1.gemfile
   - gemfiles/rails_4.2.gemfile
+  - gemfiles/rails_5.0.gemfile
+matrix:
+  exclude:
+    - rvm: 2.0.0
+      gemfile: gemfiles/rails_5.0.gemfile
+    - rvm: 2.1.5
+      gemfile: gemfiles/rails_5.0.gemfile
+    - rvm: 2.2.4
```
Comment (pschambacher) [inline .travis.yml]: That's probably useless because we're using 2.2.2

**Comment by pschambacher** [comment]
:+1:

**Comment by grosser** [comment]
:+1:

File: .travis.yml (hunk 1 above)
Comment (bquorning) [inline .travis.yml]: Fixed; now testing against 2.2.4

**Comment by bquorning** [comment]
The GitHub description says “Extra helpful tools for managing Rails 2.3 migrations”, but we don’t even test against Rails 2.3. Maybe we should discuss the purpose of this gem?

**Comment by grosser** [comment]
yeah that description is wrong :)
see email for details

**Comment by pschambacher** [comment]
Merge?

**Pull request merged**